scripts/correlator_control.py

```diff
@@ -58,17 +62,31 @@
     parser.add_argument('--testing', help="Testing mode: do not use anything that "
                         "requires a connection to the correlator "
                         "(implies dryrun) [False]", action='store_true')
+    parser.add_argument('--lstlock', help="Lock observation start to a 16s LST grid"
+                        "[True]", action='store_true',default=True)
+    parser.add_argument('--now', help="Take data now."
+                        "autofills starttime, starttime_format and starttime_scale",
+                        action='store_true')
```

Comment:
clarify that you're adding a 60s buffer in the help portion and that it's not used if starttime is set.

Reply:
done